src/main/webapp/dev/js/main/instructorCourseEnrollHandsontable.es6

```diff
@@ -0,0 +1,136 @@
+/* global Handsontable:false */
+
+/**
+ * @Global Variables
+ */
+const container = document.getElementById('spreadsheet');
+let columns = ['Section', 'Team', 'Name', 'Email', 'Comments'];
+
+/**
+ * Holds Handsontable settings, reference and other information for Spreadsheet.
+ */
+const handsontable = new Handsontable(container, {
+    rowHeaders: true,
+    colHeaders: columns,
+    columnSorting: true,
+    className: 'htCenter',
+    manualColumnResize: true,
+    sortIndicator: true,
+    maxCols: 5,
+    stretchH: 'all',
+    minSpareRows: 2,
+    manualColumnMove: true,
+    contextMenu: [
+        'row_above',
+        'row_below',
+        'remove_row',
+        'undo',
+        'redo',
+        'make_read_only',
+        'alignment',
+    ],
+});
+
+/**
+ * Updates column header order and generates a header string.
+ *
+ * Example: Change this array ['Section', 'Team', 'Name', 'Email', 'Comments']
+ * into a string = "Section | Team | Name | Email | Comments"
+ */
+function updateHeaderOrder() {
+    let headerString = '';
+    const colHeader = handsontable.getColHeader();
+    for (let itr = 0; itr < colHeader.length; itr += 1) {
+        headerString += colHeader[itr];
+        if (itr < colHeader.length - 1) {
+            headerString += '|';
+        }
+    }
+    headerString += '\n';
+    return headerString;
+}
+
+/**
+ * Updates the Student data from the spreadsheet when any of the
+ * following event occurs afterChange, afterColumnMove or afterRemoveRow.
+ *
+ * Push the output data into the textarea (used for form submission).
+ *
+ * Handles the width of column and height of row in case of overflow.
+ */
+function updateDataDump() {
+    const spreadsheetData = handsontable.getData();
+    let dataPushToTextarea = updateHeaderOrder();
+    let countEmptyColumns = 0;
+    let rowData = '';
+    for (let row = 0; row < spreadsheetData.length; row += 1) {
+        countEmptyColumns = 0; rowData = '';
+        for (let col = 0; col < spreadsheetData[row].length; col += 1) {
+            rowData += spreadsheetData[row][col] !== null ? spreadsheetData[row][col] : '';
+            if ((spreadsheetData[row][col] === '' || spreadsheetData[row][col] === null)
+             && col < spreadsheetData[row].length - 1) {
+                countEmptyColumns += 1;
+            }
+            if (col < spreadsheetData[row].length - 1) {
+                rowData += '|';
```

Comment:
This block can be rewritten as:
```javascript
if (col < spreadsheetData[row].length - 1) {
    rowData += '|';
    if (spreadsheetData[row][col] === '' || spreadsheetData[row][col] === null) {
        countEmptyColumns += 1;
    }
}
```
You can even collapse the `spreadsheetData[row][col] === '' || spreadsheetData[row][col] === null` condition to simply `!spreadsheetData[row][col]`.